rust/hyperlane-core/src/types/checkpoint.rs

```diff
@@ -206,7 +215,10 @@ impl TryFrom<&Vec<SignedCheckpointWithSigner>> for MultisigSignedCheckpoint {
 
         let signatures = sorted_signed_checkpoints
             .iter()
-            .map(|c| c.signed_checkpoint.signature)
+            .map(|c| SignatureWithSigner {
```

Comment:
Time to remove the sorting logic and comment?

Reply:
Oops yes good catch